integration-cli/utils_windows_test.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import "golang.org/x/sys/windows"
+
+// getLongPathName converts Windows short pathnames to full pathnames.
+// For example C:\Users\ADMIN~1 --> C:\Users\Administrator.
+// It is a no-op on non-Windows platforms
+func getLongPathName(path string) (string, error) {
```

Comment:
What do you think about switching on `runtime.GOOS` in the test instead of more platform files?